x-pack/legacy/plugins/index_management/public/components/mappings_editor/components/document_fields/fields/field_types/boolean_type.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import { NormalizedField, Field as FieldType } from '../../../../types';
+import { getFieldConfig } from '../../../../lib';
+import { UseField, Field, FIELD_TYPES } from '../../../../shared_imports';
+import {
+  StoreParameter,
+  IndexParameter,
+  DocValuesParameter,
+  BoostParameter,
+  NullValueParameter,
+} from '../../field_parameters';
+import { EditFieldSection, AdvancedSettingsWrapper } from '../edit_field';
+
+const getDefaultValueToggle = (param: string, field: FieldType) => {
+  switch (param) {
+    case 'boost': {
+      return field[param] !== undefined && field[param] !== getFieldConfig(param).defaultValue;
+    }
+    case 'null_value': {
+      return field.null_value !== undefined && field.null_value !== '';
+    }
+    default:
+      return false;
+  }
+};
+
+interface Props {
+  field: NormalizedField;
+}
+
+const nullValueOptions = [
+  {
+    value: '"true"',
+    text: i18n.translate('xpack.idxMgmt.mappingsEditor.stringTrueFieldDescription', {
```

Comment:
I don't think we need to translate the 4 values of the select. They should be the same in all languages, no?

Reply:
Good point :)